src/fiskaltrust.Middleware.Demo/Program.cs

```diff
@@ -74,10 +74,16 @@ private static IPOS GetPosClientForUrl(string url)
                 var uri = new Uri(url);
                 return GrpcHelper.GetClient<IPOS>(uri.Host, uri.Port);
             }
+
+            else if (url.StartsWith("rest://") || url.StartsWith("xml://"))
```

Comment:
The Url for Rest calls with JSON & XML is a plain http://
We just use rest:// for notating in the service configuration. I am not sure if we want to keep this notation in the demo as well.